src/index.html

```diff
@@ -11,7 +11,182 @@
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            required
+          >
+        </label>
+
+        <label class="block">
+          Full date of birth:
+          <input
+            type="date"
+            name="birth"
+            required
+          >
+        </label>
+
+        <label class="block">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="accept"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend>Registration</legend>
+        <label class="block">
+          E-mail:
+          <input
+            type="email"
+            name="mail"
+            placeholder="email@example.com"
+            required
+            multiple
+          >
+        </label>
+
+        <label>
+          Password:
+          <input
+            type="password"
+            name="pass"
+            required
+            autocomplete="off"
+            min="6"
+            max="20"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="block">
+          Do you love cats?
+          <label class="form-block">
+            <input
+              type="radio"
+              name="cats"
+              value="yes"
+            >Yes
+          </label>
+
+          <label class="form-block">
+            <input
+              type="radio"
+              name="cats"
+              value="no"
+            >No
+          </label>
+        </div>
+
+        <label class="block">
+          What is you favorite color?
+          <input
+            type="color"
+            name="colors"
+          >
+        </label>
+
+        <label class="block">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="sleeptime"
+            tep="1"
+            required
+          >
+        </label>
+
+        <label class="block">
+          What are your favorite brand of cars?
+          <select
+            name="cars"
+            size="4"
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="block">
+          How do you rate our work?
+          <input
+            type="range"
+            name="rate"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend>Additional info:</legend>
+        <label class="block">
+          Comments:
+          <textarea
+            name="comments"
+            cols="20"
+            rows="2"
+            minlength="50"
+            maxlength="200"
+          >
+          </textarea>
```

Comment:
```suggestion
          ></textarea>
```
to prevent this
<img width="333" alt="image" src="https://user-images.githubusercontent.com/53611910/205504162-7aeb5baf-06b7-488d-903c-ab521294d489.png">

Reply:
https://valeriiafomenko.github.io/layout_html-form/
https://valeriiafomenko.github.io/layout_html-form/report/html_report/